Products/Zuul/routers/manufacturers.py

```diff
@@ -154,11 +154,10 @@ def moveProduct(self, moveFrom, moveTarget, ids):
             "Scheduled job to move %d product(s)." % len(ids)
         )
         
-    def returnTree(self, id):
+    def returnManufacturerList(self):
         """
         return a usable tree
         """
         facade = self._getFacade()
-        data = facade.returnTree(id)                      
-        return data
-      
+        data = facade.returnManufacturerList()
```

Comment:
I don't see a `returnManufacturerList` method on the manufacture facade.  Did you forget to check that in?